### DISCUSSION TITLE
동영상 첨부 문의

### DISCUSSION THREAD

**Discussion by Epikoding** [morethanmin/morethan-log#321] [Q&A]
안녕하세요 선생님 덕분에 morethanlog를 잘 사용하고 있는 한 사용자입니다. 다름이 아니고 노션에 첨부한 동영상이 노션에서는 잘 재생이 되는데 블로그에서는 재생이 안 되는 이슈가 있어 이렇게 문의를 드리게 됐습니다.

해당 부분은 기능 구현이 어려운 것인지, 아니면 다른 방법이 있는지 궁금하네요.

**Comment by Epikoding** [comment]
자문자답입니다. 우선적으로는 유튜브에 영상을 올리고 해당 영상을 임베드하는 것으로 해결했습니다.

**Reply by dylan-kwon** [reply]
감사합니다.
유튜브는 되네요!
